charter/Charter.md

```diff
@@ -7,19 +7,19 @@
+    The TC will use CSAF CVRF 1.2 as the basis for creating OASIS Standards Track Work Products. One important consideration will be to ensure that the specification provides for sufficient interoperability to allow any consuming application to reliably process vulnerability-related remediation advisories from multiple sources without special semantic handling for each source. This includes also the provisioning, distribution and retrieving process. Knowing that there are changes which have an influence on backwards compatibility with CSAF CVRF 1.2, the TC will be attempting, where possible, to provide guidance on how to deal with these issues.
 
     The TC will develop format specifications for structured, machine-readable security vulnerability-related security advisories under the OASIS TC process, with the goal of submitting them at the appropriate time to the membership of the organization for consideration as an OASIS Standard. Other contributions will be accepted for consideration without any prejudice or restrictions and evaluated based on technical merit insofar as they conform to this charter.
 
 4. Deliverables
 
-    The TC will make substantive additions and other changes to the CVRF input specification to correct errors and evolve capabilities based on requirements and capabilities identified by OASIS TC members. The TC will rename the framework to more closely align to the primary use (e.g., Common Security Advisory Framework - CSAF). Deliverables will include a major revision of the framework. In addition to the specification deliverables, the TC may deliver supporting documentation and open source tooling on an ongoing basis in support of the TC's published standard(s). The TC expects to produce a major revision of the framework within 18 months of its first meeting.
+    The TC will make substantive additions and other changes to the CSAF CVRF 1.2 input specification to create and maintain the CSAF specification (Version 2.0 and later) to correct errors and evolve capabilities based on requirements and capabilities identified by OASIS TC members. The TC will rename the framework to more closely align to the primary use (e.g., Common Security Advisory Framework - CSAF). Deliverables will include a major revision of the framework. In addition to the specification deliverables, the TC may deliver supporting documentation and open source tooling on an ongoing basis in support of the TC's published standard(s). The TC expects to produce a major revision of the framework within 3 months of its next meeting after approving the charter.
```

Comment:
Well, I for one would (a) remove any temporal info from the last sentence in this paragraph, (b) not only include 2.0 (as it reads now), and (c) focus on the "why" we do that. Something like: 

>The TC expects to produce major revision of the framework to support the advisory sharing communities.

Reply:
Thank you @sthagen. I added the change.

Reply:
Given that change, I extended the statement about backwards compatibility to following major revisions.

Reply:
> The TC expects to produce a major revision of the framework within 3 months of its next meeting after approving the charter.

This is my (personal) hope.